### ISSUE TITLE
Upgrade Redmine image used for playbook

### ISSUE THREAD

**Issue by petiepooo** [Security-Onion-Solutions/securityonion#4102]
Please look into upgrading the docker image used for the playbook from redmine:4.1.1-passenger to redmine:4.2.1-passenger.

We use that component for other things besides the playbook, and v4.2 has the ability to add groups as issue watchers (rather than just individuals).

**Comment by defensivedepth** [comment]
What specific things are you using that redmine instance for?

**Comment by petiepooo** [comment]
We have long used Redmine running under Docker as a general issue tracking platform on our deployed sensors; a sort of client portal used to communicate and track security issues as well as other things like legal holds or documentation requests.  In SO2, I understand TheHive is meant to handle security issue tracking, but we have too much invested in Redmine to just switch over.  So we run a stock Redmine container in addition to the playbook container.
I intentionally avoided adding our custom fields, trackers, etc. to the existing playbook container just so we wouldn't have to worry about upgrades or other activities wiping out our changes.  What I ended up doing was reusing the existing so-playbook image, but overlaying a different directory as /usr/src/redmine/plugins to 1) remove the playbook plugin that we don't use, and 2) give me the opportunity to install my own plugins.  I also point it to a different backing database, of course.  The rationale is that image upgrades would happen automatically via soup, and I would have less to track.
The feature of 4.2.1 that I would like to have is the ability to add groups as watchers.  That is feature https://www.redmine.org/issues/4511 as first released in version 4.2.  While I realize that's not a feature used within the playbook, my hope was you would keep the playbook component close to the most current release much like the others.  I know I could always use another Redmine image, like the stock redmine:4.x.x-passenger series so-playbook is based on, but that would mean tracking updates myself and even more space consumed by docker images.
FWIW, I do something similar with Redis, as we've used that for years prior to SO2 as well.
